src/app/_components/Event/Card.js

```diff
@@ -32,10 +21,10 @@ function ListItem({ icon, text, textColor, fontWeight }) {
 function transformData(event) {
   const { title, organizerName, tags, priceType, eventDate, format, address, price, locationLink } = event;
   const date = new Date(eventDate);
-  const dateText = `${date.getDate()} ${MONTHS[date.getMonth()]}, ${weekDays[date.getDay()]}`;
-  const minutes = date.getMinutes().toString();
-  const hours = date.getHours().toString();
-  const timeText = `${(hours.length === 1 ? '0' : '') + hours}:${(minutes.length === 1 ? '0' : '') + minutes}`;
+  const day = date.toLocaleDateString('uk-UA', { weekday: 'long' });
```

Comment:
In addition, I would recommend moving this logic into a separate function. Maybe it will be useful in some other place.